tests/test_helpers.py

```diff
@@ -0,0 +1,14 @@
+import pandas
+
+from sharepa.helpers import pretty_print
+from sharepa.helpers import source_counts
+
+
+def test_pretty_print():
+    some_stuff = '{"Dusty": "Rhodes"}'
+    pretty_print(some_stuff)
```

Comment:
Make this test use an ```AttrDict.to_dict()``` (like from a response or something), and assert that it raises no exceptions.